corehq/apps/data_analytics/malt_generator.py

```diff
@@ -97,7 +98,6 @@ def _get_forms_queryset(self, user_id, domain_name, monthspan):
 
         return FormData.objects.exclude(
             device_id=COMMCONNECT_DEVICE_ID,
-            app_id=MISSING_APP_ID
```

Comment:
This was very bad. This makes it `AND`. It was supposed to me multiple excludes. But now we are not excluding missing-apps.